EIC-EA/EIC-community-D8
# EICNET-1114: Show breadcrumb everywhere and groep header and menu on …

…groups pages.

Tests:

* Visit all pages, and except for the homepage, there should the global breadcrumb should be visible.
* Visit the group pages and the header and group menu should be visible on all pages.

Problem:

On the discussion and wiki detail pages the group menu is not visible.
